tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -78,11 +87,11 @@ def setUpClass(cls):
         # create the Satellite's cert
         with get_connection(timeout=300) as connection:
             result = connection.run(
-                "yes | bash {} {}".format('generate-crt.sh', cls.sat6_hostname)
+                "yes | bash {} {}".format('generate-crt.sh', cert_data['sat6_hostname'])
```

Comment:
So `yes` prints continuous `y` until mentioned explicitly. Are we sure we want all the the prompts as `yes` when generate-crt.sh runs?

Reply:
That's fine I think